crates/core/Cargo.toml

```diff
@@ -15,7 +15,7 @@ version.workspace = true
 [features]
 default = []
 mainnet = []
-wasm-runtime = ["rayon"]
+wasm-runtime = []
```

Comment:
this features is never used in `crates/core` let's just remove it altogether

Reply:
fixed in a8c3d66b9b557ed4e73156615632905a247a3c04